internal/snap_printer/snap_printer_common.go

```diff
@@ -313,6 +313,20 @@ func (p *printer) expressionHasRequireOrGlobalReference(expr *js_ast.Expr) bool
 	}
 
 	switch x := expr.Data.(type) {
+	case *js_ast.EImportIdentifier:
+		ref := js_ast.FollowSymbols(p.symbols, x.Ref)
```

Comment:
This is very similar to the EIdentifier case. For imports we also have to check for name spacing aliases as well.